bin/metrics-pdns.rb

```diff
@@ -28,36 +29,23 @@
 require 'sensu-plugin'
 require 'sensu-plugin/metric/cli'
 require 'English'
+require 'socket'
 
 class PdnsGraphite < Sensu::Plugin::Metric::CLI::Graphite
-  option :dump_file,
-         description: 'name of file to dump powerDNS stats to',
-         short: '-d DUMPFILE',
-         long: '--dump DUMPFILE',
-         default: 'tmp.txt'
-
-  def output(*args)
-    return unless args.empty?
-    args[2] = Time.now.to_is if args[2].nil?
-    puts "#{`hostname -f`.chomp}-- #{args[0..2].join('\s')}"
-  end
+  option :scheme,
+         description: 'Metric naming scheme, text to prepend to metric',
+         long: '--scheme SCHEME',
+         default: "#{Socket.gethostname}.pdns"
```

Comment:
any reason to prefer Socket.gethostname over fetching it directly?

Reply:
Using `socket` is more portable and consistent with how other plugins get the hostname.